### PR TITLE
improve the help documents

### PR DESCRIPTION
**Pull request by Fish-pro**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: docs/examples/kwokctl-manage-cluster.md [hunk 1]
```diff
@@ -46,6 +46,15 @@ kubectl config use-context kwok-kwok
 
 Subsequent usage is just like any other Kubernetes cluster
 
+## Get Clusters
+
+The list of installed clusters can be obtained by `kwokctl`
```
Comment (wzshiming) [inline docs/examples/kwokctl-manage-cluster.md:51]:
```suggestion
Get the clusters managed by `kwokctl`
```

File: docs/examples/kwok-manage-nodes-and-pods.md [hunk 2]
```diff
@@ -109,6 +111,8 @@ spec:
 EOF
 ```
 
+After the creation, we can see that all the Pods are in the `Runing` state:
```
Comment (wzshiming) [inline docs/examples/kwok-manage-nodes-and-pods.md:114]:
```suggestion
After the pod is created, we can see that all the Pods are landed on that Nodes and are in the `Running` state:
```

File: docs/examples/kwokctl-snapshot.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Kwokctl Save/Restore a Cluster
 
+This document describes how to back up cluster data and restore data through `kwokctl`
```
Comment (wzshiming) [inline docs/examples/kwokctl-snapshot.md:3]:
```suggestion
This document walks you through how to save and restore a cluster with `kwokctl`
```

**Comment by Fish-pro** [comment]
Some job error，etcdctl download failed, local download is OK, I understand the network reason?
image

File: docs/contributing/development.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
-### Building `kwok` & `kwokctl`
+### Building `kwok` and `kwokctl`
 
 ``` bash
 make build
 ```
 
-On a successful build, the binaries will be located in `./bin/$(go env GOOS)/$(go env GOARCH)`
+Once successfully built, the binaries will be located in `./bin/$(go env GOOS)/$(go env GOARCH)`
```
Comment (wzshiming) [inline docs/contributing/development.md:13]:
```suggestion
On a successful build, the binaries will be located in `./bin/$(go env GOOS)/$(go env GOARCH)`
```

File: docs/examples/kwok-manage-nodes-and-pods.md [hunk 5]
```diff
@@ -61,6 +61,8 @@ status:
 EOF
 ```
 
+After the node is created, the node is in the `Ready` state:
```
Comment (wzshiming) [inline docs/examples/kwok-manage-nodes-and-pods.md:64]:
```suggestion
After the Node is created, Kwok will maintain its heartbeat and keep it in the `ready` state.
```

File: docs/examples/kwok-manage-nodes-and-pods.md [hunk 6]
```diff
@@ -61,6 +61,8 @@ status:
 EOF
 ```
 
+After the Node is created, Kwok will maintain its heartbeat and keep it in the `ready` state.
```
Comment (wzshiming) [inline docs/examples/kwok-manage-nodes-and-pods.md:64]:
```suggestion
After the Node is created, Kwok will maintain its heartbeat and keep it in the `ready` state:
```

File: docs/examples/kwok-manage-nodes-and-pods.md [hunk 7]
```diff
@@ -109,6 +111,8 @@ spec:
 EOF
 ```
 
+After the Pod is created, we can see that all the Pods are landed on that Node(s) and are in the `Running` state:
```
Comment (wzshiming) [inline docs/examples/kwok-manage-nodes-and-pods.md:114]:
```suggestion
After the Pod is created, we can see that all the Pods are landed on that Node(s) and are in the `Running` state.
```

**Comment by wzshiming** [comment]
> Some job error，etcdctl download failed, local download is OK, I understand the network reason? image

It looks like that, and I was wondering whether to add a retry mechanism to the connection timeout

**Review by wzshiming — APPROVED** [review]

**Pull request merged**